tests/system/libraries/ChromeLib.py

```diff
@@ -194,6 +197,10 @@ def getSpeechAfterKey(key) -> str:
 		"""
 		return _NvdaLib.getSpeechAfterKey(key)
 
+	@staticmethod
+	def getSpeechAndBrailleAfterKey(key) -> _Tuple[str, str]:

```

Comment:
Why not just use `_NvdaLib.getSpeechAndBrailleAfterKey` directly?

Reply:
Sure, reverted to _NvdaLib.getSpeechAndBrailleAfterKey. But I mean what's the difference - Braille doesn't seem to produce any meaningful output anyway.

Reply:
Apologies - I was under the assumption that braille had meaningful output for these commands. Feel free to not test braille in that case then.